pkg/terraform/executor_e2e_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build e2e
```

Comment:
Should e2e tests sit in `test` dir? Mixing unit tests and e2e tests?

Reply:
IMO, those are package-scoped tests, so I think it's fine if they sit in this directory. They are marked as `e2e`, as they require external dependency to pass (`terraform` binary).